README.md

```diff
@@ -64,14 +64,25 @@ In local environment, the plugin will not work properly because you will not be
 
 6. Copy migrations and templates
     ```shell
-    cp -R vendor/sylius/refund-plugin/src/Migrations/* src/Migrations
     cp -R vendor/payplug/sylius-payplug-plugin/src/Migrations/* src/Migrations
-    sed -i -- 's/Sylius\\RefundPlugin\\Migrations/DoctrineMigrations/g' src/Migrations/*
     bin/console doctrine:migrations:migrate
     mkdir -p templates/bundles/SyliusAdminBundle/
     cp -R vendor/payplug/sylius-payplug-plugin/src/Resources/views/SyliusAdminBundle/* templates/bundles/SyliusAdminBundle/
     ```
 
+    :grey_exclamation: Only for **Sylius 1.7**
+    ```shell
+    cp -R vendor/sylius/refund-plugin/src/Migrations/* src/Migrations
+    sed -i -- 's/Sylius\\RefundPlugin\\Migrations/DoctrineMigrations/g' src/Migrations/*
+    ```
+
+    :grey_exclamation: Only for **Sylius 1.8** and more, update `config/packages/doctrine_migrations.yaml` by add
+    ```yaml
+    doctrine_migrations:
+        migrations_paths:
+            'DoctrineMigrations': 'src/Migrations'
```

Comment:
By default on a SyliusStandard, we have `App\Migrations` (https://github.com/Sylius/Sylius-Standard/blob/1.8/config/packages/doctrine_migrations.yaml#L5-L6) 

Maybe you should indicate that the user can leave previous migration_paths. 

:warning: The order is important, can you please check if `App\Migrations` should stay in first position or in last position by regenerating a new migration file ?

Reply:
I modified the contents of the file to make it more understandable.

Yes, the order is important, it must be: `App\Migrations` then `DoctrineMigrations`.